pagopa/io-app
# fix: [IOPID-1359] Fix missing email on validation blocking screen

## Short description
This PR fixes the lack of the citizen's email on the blocking validation screen at startup.

<details open><summary>Example</summary>
<p>

| ❌ | ✅ | 
| - | - | 
| <img src="https://github.com/pagopa/io-app/assets/16268789/841c1d1c-52b4-4ce9-b53b-3e6784db5eff" width="250" /> | <img src="https://github.com/pagopa/io-app/assets/16268789/523a7daf-b867-41dd-8028-d34a855aa655" width="250" /> |

</p>
</details> 

## How to test
Change your email and restart the app. In the blocking validation screen you'd see your email.
